csm/core/agent/csm_agent.py

```diff
@@ -165,6 +165,9 @@ def init():
         CsmRestApi._app[const.S3_ACCESS_KEYS_SERVICE] = S3AccessKeysService(s3)
         CsmRestApi._app[const.S3_SERVER_INFO_SERVICE] = S3ServerInfoService()
 
+        # RGW
```

Comment:
Write proper comments, otherwise remove it

Reply:
Fixed